ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,143 @@
+                                <field
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                        name="./enumNames"
+                                        required="{Boolean}false"/>
+                            </enumNames>
+                            <multiSelect
+                                    granite:class="cmp-adaptiveform-dropdown__allowmultiselect"
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:orderBefore="saveValueType"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                    name="./multiSelect"
+                                    text="Allows multiple selection"
+                                    value="true"/>
+                            <multiSelect-typehint
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/hidden"
+                                    name="./multiSelect@TypeHint"
+                                    value="Boolean"/>
+                            <defaultValueSS
+                                    wrapperClass="cmp-adaptiveform-dropdown__defaultvalue"
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:orderBefore="placeholder"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                    fieldDescription="Must be one of the options."
+                                    fieldLabel="Default Value"
+                                    name="./default"/>
+                            <defaultValueMS
+                                    wrapperClass="cmp-adaptiveform-dropdown__defaultvaluemultiselect"
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:orderBefore="placeholder"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/multifield"
+                                    typeHint="String[]"
+                                    fieldDescription="Each entry must be one of the options"
+                                    fieldLabel="Default Values">
+                                <field
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                        name="./default"
+                                        required="{Boolean}false"/>
+<!--                                <default-->
```

Comment:
I'm assuming this commented code is not required now. pls remove if it's the case.

Reply:
Done